### ISSUE TITLE
JsonParseException: expecting comma separating OBJECT's entries in `swagger.json`

### ISSUE THREAD

**Issue by kkwansj** [stoicflame/enunciate#836]
Given 2 source files...
```
// FooResource.java
@Component
@Path("/{version: [vV]1}")
public class FooResource {
    @GET
    @Path("/doc")
    @Produces(MediaType.TEXT_HTML)
    public Response getDocumentation() {
        String path = this.servletContext.getRealPath("/html");
        File file = new File(path + File.separatorChar + "index-v1.html");
        return Response.ok().entity(file).build();
    }
    ...
```
and
```
// BarResource.java
@Component
@Path("/{version: [vV]2}")
public class BarResource {
    @GET
    @Path("/doc")
    @Produces(MediaType.TEXT_HTML)
    public Response getDocumentation() {
        String path = this.servletContext.getRealPath("/html");
        File file = new File(path + File.separatorChar + "index-v2.html");
        return Response.ok().entity(file).build();
    }
    ...
```
Enunciate's Swagger module generates the following JSON (swagger.json) which is missing `,` after line 23:
```
 1  "\/{version}/doc":{
 2    "get":{
 3      "tags":[
 4        "FooResource"
 5      ],
 6      "description":"",
 7      "operationId":"getDocumentation",
 8      "produces":[
 9        "text/html"
10      ],
11      "parameters":[
12
13      ],
14      "responses":{
15        "200":{
16          "schema":{
17            "description":"",
18            "type":"string"
19          },
20          "description":""
21        }
22      }
23    }
24    "get":{
25      "tags":[
26        "BarResource"
27      ],
28      "description":"",
29      "operationId":"getDocumentation2",
30      "produces":[
31        "text/html"
32      ],
33      "parameters":[
34
35      ],
36      "responses":{
37        "200":{
38          "schema":{
39            "description":"",
40            "type":"string"
41          },
42          "description":""
43        }
44      }
45    }
46  }
47  ,
```
resulting in the error:
```
[ERROR] Failed to execute goal com.webcohesion.enunciate:enunciate-maven-plugin:2.10.1:assemble (default) on project foobar: com.fasterxml.jackson.core.JsonParseException: Unexpected character ('"' (code 34)): was expecting comma to separate OBJECT entries
[ERROR]  at [Source: java.io.FileReader@12f93a53; line: xxxx, column: 8]: OnError while emitting onNext value: com.webcohesion.enunciate.EnunciateContext.class
```
This appears to only be present with Enunciate version 2.10.x

**Comment by stoicflame** [comment]
Fixed at b4fdbedf56b01102c9cb1367c348ce23792a111f.

**Comment by stoicflame** [comment]
[Enunciate 2.11 has been released](https://github.com/stoicflame/enunciate/releases/tag/v2.11.0).
